financial-connections/src/main/java/com/stripe/android/financialconnections/features/accountpicker/AccountPickerScreen.kt

```diff
@@ -110,51 +103,8 @@ private fun AccountPickerContent(
     onLoadAccountsAgain: () -> Unit,
     onCloseFromErrorClick: (Throwable) -> Unit
 ) {
-    val scope = rememberCoroutineScope()
     val lazyListState = rememberLazyListState()
-    ModalBottomSheetLayout(
```

Comment:
We can remove a bunch of `ModalBottomSheetLayout` usages within panes.

Reply:
this makes me happy